modules/tasks/tasks/src/main/resources/webapp/css/tasks.css

```diff
@@ -254,6 +254,7 @@ manipulation-sorter .manipulation .remove:before {
 
 .tasks .map-field {
     width: 30%;
+    overflow-x: auto;
```

Comment:
The Tasks module currently uses auto-expandable inputs (at least for raw text values). Could you verify whether we could somehow auto-expand the inputs for bubbles as well? This would look way better than adding a scroll.